tests/anderson_test.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from absl.testing import absltest
+from absl.testing import parameterized
+
+import jax
+import jax.numpy as jnp
+from jax import test_util as jtu
+from jax.tree_util import tree_map, tree_all
+from jax.test_util import check_grads
+
+from jaxopt.tree_util import tree_l2_norm, tree_scalar_mul, tree_sub
+from jaxopt import objective
+from jaxopt import AndersonAcceleration
+
+import numpy as np
+import scipy
+from sklearn import datasets
+
+
+class AndersonAccelerationTest(jtu.JaxTestCase):
+  
+  def test_geometric_decay(self):
+    """ Test convergence for geometric progression with common ratio r < 1.
+    """
```

Comment:
No need for the line return here since the docstring fits on one line (same below).